SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimAnalyzer.cc

```diff
@@ -291,7 +1044,173 @@ EcalEBTrigPrimAnalyzer::endJob(){
     hRH_iphiVsieta_->Write();
     hTP_iphiVsieta_fullrange_->Write();
     hRH_iphiVsieta_fullrange_->Write();
-
+    hCluTPoverRechit_->Write();
+    hCluTPvsRechit_->Write();
   }
+
+  h2_recEle_vs_Gen_size_->Write();
+  h2_cluTP_vs_Gen_size_ ->Write();
+  h2_cluRH_vs_Gen_size_ ->Write();
+
+
+
+
 }
+
+
+
+std::vector<SimpleCluster>  EcalEBTrigPrimAnalyzer::makeCluster (std::vector<SimpleCaloHit2> &  hitCollection, int dEta, int dPhi ) {
+
+
+   if (debug_) std::cout << "  makeCluster  input collection size " << hitCollection.size() << std::endl;
+
+   std::vector<SimpleCluster> clusters;
+   while (true) {
+     
+     SimpleCaloHit2 centerhit(0);      
+     for (unsigned int iRH=0;iRH<hitCollection.size();++iRH) {
+       SimpleCaloHit2  hit = hitCollection[iRH];  
+
+       float energy = hit.et()/sin(hit.position().theta());
+
+	//if ( energy < 0.2 ) continue;
```

Comment:
Delete commented out code, or justify with a comment why you still need to keep it